CHANGELOG.rst

```diff
@@ -4,6 +4,20 @@ Changelog
 
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
+1.12.2 (2021-11-02)
+-------------------
+
+**Added**
+
+**Fixed**
+
+**Dependencies**
+
+* `vaadin:7.7.17` ->  `vaadin:7.7.28`
```

Comment:
* com.vaadin 7.7.17 -> 7.7.28 (addresses CVE-2021-37714)

Reply:
good point. I will change it